src/Components/CardsContainer.js

```diff
@@ -0,0 +1,29 @@
+import SingleCard from "./SingleCard";
+import CardGroup from "react-bootstrap/CardGroup";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import "./CardsContainer.css";
+
+function CardsContainer() {
+  return (
+    <Container className="cardsContainer">
+      <Row>
+        <Col>
+          <SingleCard />
+        </Col>
+        <Col>
+          <SingleCard />
+        </Col>
+        <Col>
+          <SingleCard />
+        </Col>
+        <Col>
+          <SingleCard />
+        </Col>
+      </Row>
```

Comment:
Shouldn't we use a CardGroup here? Since Bootstrap automatically adapts to the screen size and to the number of cards

Reply:
We'll talk about this in the next meeting. I'll merge this PR as is for now.

Reply:
I tried to use the CardGroup but the inner parts of the cards did not get rounded, I will see if we can use it